alchemiscale/base/client.py

```diff
@@ -119,9 +118,9 @@ def _post_resource(self, resource, data):
         return resp.json()
 
     def get_info(self):
-        return self._get_resource("/info", params={}, return_gufe=False)
+        return self._get_resource("/info")
 
     @_use_token
     def _api_check(self):
         # Check if the API is up and running and can reach services
-        self._get_resource("/check", params={}, return_gufe=False)
```

Comment:
I think this is a good idea moving the gufe logic outside this method, I kept getting caught out by it.

Reply:
Yeah, we ended up disabling the behavior 90% of the time, so seemed right to just drop it.